src/applications/personalization/profile/actions/paymentInformation.js

```diff
@@ -170,10 +169,11 @@ export function saveCNPPaymentInformation({
 
     if (response.error || response.errors) {
       const errors = response.error?.errors || [];
```

Comment:
I'm not 100% sure to be honest. 

This is how it was from a past engineer, so I didn't change it, but that would make sense that it should use `response.errors` instead of an empty array.

I'll reevaluate and see if I can determine why it is done this way.